packages/validator/src/lib/psi/PsiResult.ts

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+export type LighthouseCategoryName =
+    'accessibility' | 'best-practices' | 'performance' | 'pwa' | 'seo';
+export type LighthouseEmulatedFormFactor = 'desktop' | 'mobile';
```

Comment:
We'd like websites to work well on all form factors, but given this is a tool for generating TWAs, do we have a solid reason for caring about how they perform/look on desktop?

Reply:
This is just mapping the output for the API call and the valid values for the fields.